picturae_importer.py

```diff
@@ -969,6 +981,13 @@ def run_all_methods(self):
                         self-explanatory function, will run all methods in class in sequential manner"""
         # code to create test images for test image uploads
         # setting directory
+
```

Comment:
backup copy of csv, to preserve original booleans, so that user won't have to run all the way back to wrangler to re-process, but still have the safety of not accidentally double uploading data.